api/src/controllers/schema.ts

```diff
@@ -67,23 +62,25 @@ const schemaMultipartHandler: RequestHandler = (req, res, next) => {
 
 			if (mimeType === 'application/json') {
 				try {
-					uploadedSnapshot = parseJSON(uploadedString);
+					upload = parseJSON(uploadedString);
 				} catch (err: any) {
 					logger.warn(err);
 					throw new InvalidPayloadException('Invalid JSON schema snapshot');
```

Comment:
We probably need to update this exception message depending on where it's used (eg. `/diff` should return "invalid snapshot" like how it currently is, but `/apply` should return "invalid diff" instead for clarity). 

Maybe we can use a wrapper like `getSchemaMultipartHandler<Snapshot>({errorMessage: 'Invalid JSON schema snapshot'})`? Very rough initial idea here, not sure if it's overkill. (the generic `<Snapshot>` relates to the other review comment)

Reply:
That seems indeed a bit overkill 🤔 I might go with a generic error message then. This seems appropriate since here every error is thrown regarding a invalid JSON or YAML. Specific errors regarding the diff or the snapshot are handled in the schema service, so it might be ok here to be generic

Reply:
Makes perfect sense, and the generic errors LGTM! 👍